modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -194,8 +194,14 @@ Because the following endpoints use mutual TLS authentication to establish the c
 | Port | Protocol | Hostnames | Description
 .2+| 443 .2+| AMQP over WebSockets | US control plane: `transport-layer.prod.cloudhub.io` | Runtime Fabric message broker for interaction with the control plane.
 | EU control plane: `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the control plane.
-.2+| 5044 .2+| TCP (Lumberjack) | US control plane: `dias-ingestor-nginx.prod.cloudhub.io` | Anypoint Monitoring agent for Runtime Fabric.
+.2+| 5044 (deprecated) .2+| TCP (Lumberjack) | US control plane: `dias-ingestor-nginx.prod.cloudhub.io` | Anypoint Monitoring agent for Runtime Fabric.
+
+*This port and hostname are deprecated in Anypoint Runtime Fabric, version 1.8.48.* 
+
+If you are using a previous version of Anypoint Runtime Fabric you must add this port and hostname to your allow list. If you are using a newer version, use the port and hostname specified below.
```

Comment:
@sleslie23 Is there a way to place this note so that it applies for both the US and EU control plane endpoints?
Otherwise LGTM

Reply:
I can add something like "This is applicable to endpoints in both the US and EU clouds." It's better to avoid trying change the format of the table rows.